static/src/stylesheets/_vars.scss

```diff
@@ -120,6 +120,8 @@ $frontend-palette: (
 
     paidfor-background:     guss-colour(news-support-3, $pasteup-palette),
 
+    special-report-accent:  #fce800,
```

Comment:
Is this an 'official' Guardian colour? It isn't on http://guardian.github.io/pasteup-palette/demo/ ...